spec/models/miq_server/worker_monitor_spec.rb

```diff
@@ -351,48 +351,60 @@
         end
       end
 
-      context "with worker that is using a lot of memory" do
+      context "threshold validation" do
+        let(:worker) { FactoryGirl.create(:miq_worker, :miq_server_id => server.id, :pid => 42) }
+        let(:server) { @miq_server }
+
         before(:each) do
-          @worker1 = FactoryGirl.create(:miq_worker, :miq_server_id => @miq_server.id, :memory_usage => 2.gigabytes, :pid => 42)
-          allow_any_instance_of(MiqServer).to receive(:get_time_threshold).and_return(2.minutes)
-          allow_any_instance_of(MiqServer).to receive(:get_memory_threshold).and_return(500.megabytes)
-          allow_any_instance_of(MiqServer).to receive(:get_restart_interval).and_return(0.hours)
```

Comment:
Haha!  Nice emoji variety.  Funny story:  I looked at those tests and couldn't understand why it was using `allow_any_instance_of`.  I was shocked and overly excited when the tests still passed using the `server` instance we already had!